Kernel/Graphics/VirtIOGPU/DisplayConnector.cpp

```diff
@@ -96,16 +96,9 @@ ErrorOr<void> VirtIODisplayConnector::set_safe_mode_setting()
     return set_mode_setting(safe_mode_setting);
```

Comment:
Maybe add a note in the code, why we are not double buffering anymore

Reply:
Done.